django/db/models/functions/comparison.py

```diff
@@ -36,6 +37,11 @@ def as_sqlite(self, compiler, connection, **extra_context):
             return super().as_sql(
                 compiler, connection, template=template, **extra_context
             )
+        elif output_type == "JSONField":
+            template = "JSON(%(expressions)s)"
+            return super().as_sql(
+                compiler, connection, template=template, **extra_context
+            )
```

Comment:
Could we extract the changes in `Cast()`, along with the corresponding test, into a separate commit please?

Reply:
Done :heavy_check_mark: